### DISCUSSION TITLE
Empezamos en Python!

### DISCUSSION THREAD

**Discussion by mchalin** [CodeSystem2022/ProyectoIntegrador_UTN_BuenosAires#6] [Announcements]
Gente, se creo proyecto para ir colgando los avances del Proyecto Integrador, dense una vuelta y vayan asignándose tareas!

**Comment by vmgith** [comment]
18/5 Reunion del equipo para organizar el comienzo del proyecto Integrador

**Comment by mchalin** [comment]
Buenísimo! A q hora?
